src/pages/plants/PlantCard.tsx

```diff
@@ -4,7 +4,7 @@ const PlantCard = ({ image, name, onClick }: CardType) => {
   return (
     <div onClick={onClick}>
       <img className="w-[200px] h-[180px]" src={image} alt={name} />
-      <p className="text-center pt-5 text-lg">{name}</p>
+      <p className="w-full text-center pt-1 text-lg inline-block">{name}</p>
```

Comment:
why did you add w-full and inline block?
because <p> tag is block element, so if you want inline-block then use <span> tag